data/human/free worlds 4 epilogue.txt

```diff
@@ -45,7 +45,7 @@ mission "FW Epilogue: Ijs and Katya"
 			
 			label game
 			`	"Whoops," says Ijs, "now you've done it, there goes the fourth wall."`
-			`	Katya laughs. "If you're so eager to explore the rest of the galaxy, <first>, maybe you should volunteer to help out with creating the stories to populate it. This is an open source game after all, and it would be a sad thing for it to remain the product of only one person's imagination."`
+			`	Katya laughs. "No. There are more missions and campaigns after this one, however, there also are parts of the game that have not been implemented yet. If you're so eager to explore the rest of the galaxy, <first>, maybe you should volunteer to help out with creating the stories to populate it. This is an open source game after all, and it would be a sad thing for it to remain the product of only one person's imagination."`
```

Comment:
```suggestion
			`	Katya laughs. "Well, there's already lots beyond human space to see, and I'm sure you can find more things to do here as well as out there. If you're so eager to explore even more of the galaxy or the adventures within it beyond that, <first>, maybe you should volunteer to help out with creating the stories to populate it. This is an open source game after all, and it would be a sad thing for it to remain the product of only one person's imagination."`
```

Reply:
This sounds good, except I would turn `even more` into just `more`.

Reply:
Yes, this looks good. I think either `even more` _or_ `more` work, just with different emphasis.

Reply:
```suggestion
			`	Katya laughs. "Well, there's already lots beyond human space to see, and I'm sure you can find more things to do out there as well as closer to home. If you're so eager to explore more of the galaxy, <first>, or to discover what new adventures it has to offer, maybe you should volunteer to help out with creating the stories to populate it. This is an open source game after all, and it relies on people like you to bring it life."`
```

I feel like the '...explore even more of the galaxy or the adventures within it beyond that...' section was a bit awkward so I have attempted a rewrite of that. Also a couple of other small changes, particularly changing the 'one person's imagination' bit as that is definitely no longer the case.

Reply:
Yea, I'm liking that version @LazerLit  :)

Reply:
Yes, that version also more adequately reflects the current dev community.